plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -501,12 +501,13 @@ const fetchPluginVersions = async ({createNode, reporter, firstReleases}) => {
 const fetchPluginHealthScore = async ({createNode, reporter}) => {
     const sectionActivity = reporter.activityTimer('fetch plugin health score');
     sectionActivity.start();
-    const url = 'https://plugin-health.jenkins.io/api/scores';
-    const json = await requestGET({url, reporter});
-    for (const pluginName of Object.keys(json.plugins)) {
-        const data = json.plugins[pluginName];
+    const url = 'http://localhost:8080/api/scores';
+    const {plugins, statistics} = await requestGET({url, reporter});
+    for (const pluginName of Object.keys(plugins)) {
+        const {value, details} = plugins[pluginName];
         createNode({
-            ...data,
+            value,
+            details,
```

Comment:
@halkeye I might need your help here. 
I don't understand why but the `details` (with https://github.com/jenkins-infra/plugin-health-scoring/pull/343, it's an array) is never put in the node.
That's the same for the `statistics` in fact. Its value is never deconstruct into the node under the graphql explorer..

Reply:
I'm not sure, maybe if you are changing structure maybe you have something cached? you can run `npx gatsby clean` in the plugins/plugin-site directory, just to be certain. I know for gatsby graphql schemas are inferred, so if one entry is an array and one is an object, it would get confused.
you can force graphql schema with createSchemaCustomization

you can also take a look at ./plugins/plugin-site/schema.graphql to see what the schema actually looks like, to confirm if there's anything.

Without being able to dig into the data though, I can't really do much investigation.

maybe swap out the createNode with console.log to confirm its trying to create with the right values, and nothing is null/undefined that you expect.

Reply:
I'll try the gatsby clean but when I added a key `foo` into the object I could see it and I did use `console.log` and could see the `details` array. I'll try a few more things...

Reply:
ooh, can you save your local /api/scores to a gist, and I'll just use that to poke around.

Reply:
I'll do that. Thank you for your help.

Reply:
here you go: https://gist.github.com/alecharp/4c0dab7873c98dff441ecbf5089d6577. Thank you.

However, it seems that the `npx gatsby clean` make it work. I can see the `details` array in the explorer as well as in the schema. You seems to have fixed the problem without having to reproduce it even. Nice!